stock_picking_invoice_link/models/stock_move.py

```diff
@@ -4,22 +4,12 @@
 # Copyright 2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    # Provide this field for backwards compatibility
-    invoice_line_ids = fields.Many2many(
```

Comment:
@pedrobaeza Should a migration script be provided here or all fields will come already computed from previous version?

Reply:
No, the field was preserved in previous versions because the change was in the middle of the version, and there can be other modules that inherits this one using that field. Now it's totally not needed.